fastlane/lib/fastlane/action_collector.rb

```diff
@@ -15,7 +26,28 @@ def show_message
     end
 
     def determine_version(name)
-      super(name) || Fastlane::VERSION
+      self.class.determine_version(name)
+    end
+
+    # e.g.
+    #   :gym
+    #   :xcversion
+    #   "fastlane-plugin-my_plugin/xcversion"
+    def self.determine_version(name)
+      return super(name) if super(name)
```

Comment:
I wonder if there is a Ruby operator for this, since it's quite common. I updated the code 👍